From 85d2e0e12370a20ae54118fb889ab422e2557213 Mon Sep 17 00:00:00 2001
From: Arjan Molenaar <gaphor@gmail.com>
Date: Tue, 11 May 2021 22:49:55 +0200
Subject: [PATCH] Fix Gtk.ListStore item insertion for GTK 4.1

Fixes #467.
---
 gi/overrides/Gtk.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/gi/overrides/Gtk.py b/gi/overrides/Gtk.py
index 7739751e..de6b4465 100644
--- a/gi/overrides/Gtk.py
+++ b/gi/overrides/Gtk.py
@@ -998,7 +998,10 @@ class ListStore(Gtk.ListStore, TreeModel, TreeSortable):
     def _do_insert(self, position, row):
         if row is not None:
             row, columns = self._convert_row(row)
-            treeiter = self.insert_with_valuesv(position, columns, row)
+            if hasattr(self, "insert_with_valuesv"):
+                treeiter = self.insert_with_valuesv(position, columns, row)
+            else:
+                treeiter = self.insert_with_values(position, columns, row)
         else:
             treeiter = Gtk.ListStore.insert(self, position)
 
-- 
GitLab

