diff --git a/data/org.gabmus.hydrapaper.appdata.xml.in b/data/org.gabmus.hydrapaper.appdata.xml.in
index 4fdd6e3..96f747d 100644
--- a/data/org.gabmus.hydrapaper.appdata.xml.in
+++ b/data/org.gabmus.hydrapaper.appdata.xml.in
@@ -18,22 +18,22 @@
     <launchable type="desktop-id">org.gabmus.hydrapaper.desktop</launchable>
     <screenshots>
         <screenshot type="default">
-            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/website/screenshots/mainwindow.png</image>
+            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/static/screenshots/mainwindow.png</image>
         </screenshot>
         <screenshot>
-            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/website/screenshots/folders.png</image>
+            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/static/screenshots/folders.png</image>
         </screenshot>
         <screenshot>
-            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/website/screenshots/favorites.png</image>
+            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/static/screenshots/favorites.png</image>
         </screenshot>
         <screenshot>
-            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/website/screenshots/random.png</image>
+            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/static/screenshots/random.png</image>
         </screenshot>
         <screenshot>
-            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/website/screenshots/preferences.png</image>
+            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/static/screenshots/preferences.png</image>
         </screenshot>
         <screenshot>
-            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/website/screenshots/modes.png</image>
+            <image>https://gitlab.gnome.org/gabmus/hydrapaper/raw/website/static/screenshots/modes.png</image>
         </screenshot>
     </screenshots>
     <url type="homepage">https://hydrapaper.gabmus.org</url>
